staging/src/k8s.io/apimachinery/pkg/conversion/deep_equal.go

```diff
@@ -34,3 +34,16 @@ func EqualitiesOrDie(funcs ...interface{}) Equalities {
 	}
 	return e
 }
+
```

Comment:
comment above Equalities is wrong

Reply:
Hi Stefan, to my understanding the comment is correct. Is this not an implementation of a shallow copy for a map? Thanks